website/server/libs/slack.js

```diff
@@ -52,12 +54,17 @@ function sendFlagNotification ({
     authorName = `${message.user} - ${authorEmail} - ${message.uuid}`;
   }
 
+  let date = moment(message.timestamp).format('YYYY-MM-DD');
```

Comment:
you should only call moment once, also for variable that do not change it's better to use `const` instead of `let`. I think you should be able to condense everything in one line here

```
const timestamp = `${moment(message.timestamp).utc().format('YYYY-MM-DD HH:mm')} UTC` 
```